### DISCUSSION TITLE
archlinux package

### DISCUSSION THREAD

**Discussion by pricechrispy** [FoobarOy/foomuuri#39] [Show and tell]
Hello, I now use this in archlinux and it's great. I created a package https://aur.archlinux.org/packages/foomuuri that can be added to your wiki https://github.com/FoobarOy/foomuuri/wiki/Getting-Started . Thank you for putting your time an energy into this. I can't wait to see how it develops further!

**Comment by kimheino** [comment]
Thanks for adding Foomuuri package to archlinux.

I have never used aur/PKGBUILD/makepkg. Can you give me example commands that I can add to Getting-Started wiki page?

**Reply by pricechrispy** [reply]
Sure! I think It would be best to point people to the archlinux wiki as that will be maintained long term: https://wiki.archlinux.org/title/Arch_User_Repository#Installing_and_upgrading_packages

Adapting those, it could be something like:
1. `git clone https://aur.archlinux.org/foomuuri.git`
2. `cd foomuuri`
3. `makepkg`
4. `pacman -U foomuuri-0.23-1-x86_64.pkg.tar.zst`

Though, many people use unofficial build helpers like Yay that are "not supported" (https://wiki.archlinux.org/title/AUR_helpers).

Those people could just run: `yay -S foomuuri`

**Comment by kimheino** [comment]
Wiki updated: https://github.com/FoobarOy/foomuuri/wiki/Getting-Started#arch-linux

**Reply by pricechrispy** [reply]
Awesome! Do you have concerns if I package a git version as well? This is common with many packages.

One package for releases for stability.
One for development via git default branch for access to newer features.

I see you have both 0.24 and 0.25 in the upcoming changelog.

I'm wondering, how would you feel if a package builds off main for the latest features? Is there a different preferred way to package upcoming beta releases (dev/beta branch, pending version tag/branch, etc.)?

**Comment by kimheino** [comment]
> Awesome! Do you have concerns if I package a git version as well? This is common with many packages.

Feel free to do so.  Git versions should be quite stable anyway. 

> I see you have both 0.24 and 0.25 in the upcoming changelog.

That was a mistake. Fixed now. Next version will be 0.24, there is no 0.25. Currently I try to do release every 2-3 months so next one will be released at end of month.

> I'm wondering, how would you feel if a package builds off main for the latest features? 

"main" is the correct and only branch. I don't have have any dev/beta branches at the moment.

**Reply by pricechrispy** [reply]
Awesome, packaging it here now: https://aur.archlinux.org/packages/foomuuri-git
